api/app/src/main/kotlin/packit/model/dto/RolesAndUsers.kt

```diff
@@ -0,0 +1,14 @@
+package packit.model.dto
+
+import packit.model.Role
+import packit.model.User
+
+data class RolesAndUsers(
+    val roles: List<Role> = listOf(),
+    val users: List<User> = listOf()
+)
+
+data class RolesAndUsersWithPermissionsDto(
+    val roles: List<RoleDto> = listOf(),
+    val users: List<UserWithPermissions> = listOf()
```

Comment:
Same comment about "users are roles" applies here I guess, isn't this just a list of roles? Does the roles list include usernames?

Reply:
as per above comment.